week1/task_学生成绩管理系统0.1.cpp

```diff
@@ -0,0 +1,62 @@
+#include <stdio.h> 
+int Not (float score[],int n);
+void Datescore(float score[],int n);
+int main()
+{
+	float score[40];
+	int n,i,a;
+	printf("please input the number of the students(n<=40):");
+	scanf("%d",&n);
+	printf("please input the score of the students:\n");
+	for (i=0;i<n;i++)
+	{
+		scanf("%f",&score[i]);
+	}
+	a=Not (score,n);

```

Comment:
1. add an empty line after code block
2. there should be spaces between assignment operator but no space is allowed between function name and (
```suggestion
	a = not(score,n);
```